docs/src/examples/Thermal_Generation_Dispatch_Example.jl

```diff
@@ -0,0 +1,153 @@
+# # Thermal Generation Dispatch Example
+
+#md # [![](https://img.shields.io/badge/GitHub-100000?style=for-the-badge&logo=github&logoColor=white)](@__REPO_ROOT_URL__/examples/Thermal_Generation_Dispatch_Example.jl)
+
+# This example illustrates the sensitivity analysis of thermal generation dispatch problem.
+
+# This problem can be described as the choice of thermal generation `g` given a demand `d`, a price for thermal generation `c` and a penalty price `c_{ϕ}` for any demand not attended ϕ.
+
+# ```math
+# \begin{split}
+# \begin{array} {ll}
+# \mbox{minimize} & \sum_{i=1}^{N} c_{i} g_{i} + c_{\phi} \phi \\
+# \mbox{s.t.} & g_{i} \ge 0 \quad i=1..N  \\
+#             & g_{i} \le G_{i} \quad i=1..N  \\
+#             & \sum_{i=1}^{N} g_{i} + \phi = d\\
+# \end{array}
+# \end{split}
+# ```
+# where
+# - `G_{i}` is the maximum possible generation for a thermal generator `i`
+
+# ## Define and solve the Thermal Dispatch Problem
+
+# First, import the libraries.
+
+using Test
+using LinearAlgebra
+using DiffOpt
+using Clp
+using MathOptInterface
+using JuMP
+using Plots
+const MOI = MathOptInterface
```

Comment:
```suggestion
using JuMP
import DiffOpt
import LinearAlgebra: dot
import GLPK
import MathOptInterface
import Plots
const MOI = MathOptInterface
```

Reply:
We are actually using GLPK instead of Clp on master.
We should strive to use import for packages other than JuMP to make it clear where a method comes from.